game/scripts/vscripts/abilities/fountain_attack.lua

```diff
@@ -32,7 +32,9 @@ function modifier_fountain_attack:GetModifierAura()
 end
 
 function modifier_fountain_attack:GetAuraRadius()
-  return self.trigger:GetBoundingMaxs():Length2D()
+  local caster = self:GetCaster()
+  local distance = caster:GetAbsOrigin() + self.trigger:GetAbsOrigin()
+  return self.trigger:GetBoundingMaxs():Length2D() + distance
```

Comment:
This function doesn't make any sense.

It takes caster location + trigger location, which is some spot very far away (`-` would give the distance as a vector, + is the two combined)
then it takes the bounding max's, which are the largest extremities of the given zone, calculates the 2d length of that vector, and adds that float value to the vector position...

Reply:
@Chronophylos i dunno. it works though.

Reply:
What are you trying to do here?

Reply:
that's the bounding box for target selection for the fountain lazor

Reply:
something i changed broke it. this is what fixed it.

Reply:
if i had to guess, the trigger box is no longer at the exact same spot as the entity. `return self.trigger:GetBoundingMaxs():Length2D()` seems correct to me.

Reply:
ok fine i'll see what happens i guess....